deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
hm, looks like if you have a system llvm-config that happens to be on the path, then something in the makefiles is picking it up here and trying to run it even if USE_SYSTEM_LLVM isn't set...